src/workflowEngine/WorkflowWorker.js

```diff
@@ -119,9 +121,18 @@ class WorkflowWorker {
       else if (!Array.isArray(vars[varName])) vars[varName] = [vars[varName]];
 
       vars[varName].push(value);
+      variableName = varName;
+    } else {
+      vars[name] = value;
+    }
+
+    if (variableName.startsWith('$$')) {
+      dbStorage.variables.put({
+        value,
+        name: variableName,
```

Comment:
My workflow keeps running repeatedly while it should be once per day (by checking variables in workflow). Then I found in Storage tab, the variables are prefixed with duplicated "$$". So, I guess this is the cause.

In previous code, the variable's name was sliced: https://github.com/AutomaApp/automa/pull/1522/files#diff-e2be43c3a6d1b35564d0d2980d83866ffef90cca2e66eded0a5ac7b47db31a08L509

Reply:
You're right, will fix in the next update